google/cloud/bigtable/tests/instance_admin_integration_test.cc

```diff
@@ -195,57 +195,66 @@ TEST_F(InstanceAdminIntegrationTest, DeleteInstancesTest) {
 
 /// @test Verify that InstanceAdmin::ListClusters works as expected.
 TEST_F(InstanceAdminIntegrationTest, ListClustersTest) {
-  // The emulator does not support cluster operations.
-  if (UsingCloudBigtableEmulator()) {
-    return;
-  }
-  std::string id = "list-clusters-test";
-  bigtable::InstanceId instance_id(id);
-  bigtable::DisplayName display_name(id);
-  std::vector<std::pair<std::string, bigtable::ClusterConfig>> clusters_config;
-  clusters_config.push_back(std::make_pair(
-      id + "-cluster1", bigtable::ClusterConfig("us-central1-f", 0,
-                                                bigtable::ClusterConfig::HDD)));
-  auto instance_config =
-      bigtable::InstanceConfig(instance_id, display_name, clusters_config)
-          .set_type(bigtable::InstanceConfig::DEVELOPMENT);
-  auto instance_details =
-      instance_admin_->CreateInstance(instance_config).get();
-
-  // Create clusters in an instance
-  // TODO(#422) - Implement InstanceAdmin::CreateCluster
+  std::string id =
+      "it-" + bigtable::testing::Sample(generator_, 8,
+                                        "abcdefghijklmnopqrstuvwxyz0123456789");
+  auto config = IntegrationTestConfig(id);
+  auto instance = instance_admin_->CreateInstance(config);
+  ASSERT_THAT(instance.get().name(), HasSubstr(id));
 
-  // TODO(#418) - create an instance and test that its cluster is returned here.
   auto clusters = instance_admin_->ListClusters(id);
-  for (auto const& i : clusters) {
-    auto const npos = std::string::npos;
-    EXPECT_NE(npos, i.name().find(instance_admin_->project_name()));
+  for (auto const& cluster : clusters) {
+    EXPECT_NE(std::string::npos,
+              cluster.name().find(instance_admin_->project_name()));
+    EXPECT_NE(std::string::npos, cluster.name().find(id));
   }
   EXPECT_FALSE(clusters.empty());
 
   instance_admin_->DeleteInstance(id);
 }
 
+/// @test Verify that default InstanceAdmin::ListClusters works as expected.
+TEST_F(InstanceAdminIntegrationTest, ListAllClustersTest) {
+  std::string id1 =
+      "it-" + bigtable::testing::Sample(generator_, 8,
+                                        "abcdefghijklmnopqrstuvwxyz0123456789");
+  std::string id2 =
+      "it-" + bigtable::testing::Sample(generator_, 8,
+                                        "abcdefghijklmnopqrstuvwxyz0123456789");
+
+  auto instance_config1 = IntegrationTestConfig(
+      id1, "us-central1-c", bigtable::InstanceConfig::PRODUCTION, 3);
+  auto instance_config2 = IntegrationTestConfig(
+      id2, "us-central1-f", bigtable::InstanceConfig::PRODUCTION, 3);
+  auto instance1 = instance_admin_->CreateInstance(instance_config1);
+  // Wait for instance creation
+  ASSERT_THAT(instance1.get().name(), HasSubstr(id1));
+  auto instance2 = instance_admin_->CreateInstance(instance_config2);
```

Comment:
Not that it matters, but if you put both `CreateInstance()` requests together, that should be "faster" (the instances can be created in parallel).